### ISSUE TITLE
DS's can view tensorflow graphs of their trained models (uncouth way)

### ISSUE THREAD

**Issue by darshanraju** [morphware/service#49]
Magnet URI's to the loss graphs should also be stored/emitted from the contracts
